desk/app/groups-ui.hoon

```diff
@@ -238,4 +254,28 @@
   ?+  wire  !!
     [%build ~]  cor
   ==
+++  preload-contacts
+  =+  .^(chat-running=? (scry %gu %chat /$))
+  =?  cor  chat-running
+    =+  .^  [dms=(map ship dm:c) *]
+        (scry %gx %chat /full/noun)
+      ==
```

Comment:
Usually you would align `==` with the opening rune, but perhaps here it is ok? @Fang-

Reply:
```suggestion
    =+  .^  [dms=(map ship dm:c) *]
      (scry %gx %chat /full/noun)
    ==
```

Reply:
@arthyn the `==` is closing the `.^`, so aligning with the opening rune here would look like such:

```suggestion
    =+  .^  [dms=(map ship dm:c) *]
          (scry %gx %chat /full/noun)
        ==
```